src/components/tabs/demoDynamicTabs/index.html

```diff
@@ -16,13 +16,25 @@
 
   <form ng-submit="addTab(tTitle,tContent)" flex>
     <div layout="row" layout-sm="column" layout-margin layout-align="left center">
-      <md-text-float label="Active Index" ng-model="selectedIndex" disabled></md-text-float>
-      <md-text-float label="Active Title" ng-model="tabs[selectedIndex].title"></md-text-float>
+      <md-input-container>
+        <label>Active Index</label>
+        <input type="text" ng-model="selectedIndex">
```

Comment:
Thanks @gkalpak 

I added disabled attribute and also when i was checking it on the browsers i found bug that on IE11 disabled input hadn't dotted background, so i fixed it also. I hope it'll be helpful.